help_data.yaml

```diff
@@ -61,6 +61,10 @@ livepatch:
       more about Ubuntu Kernel Livepatch service at
       https://ubuntu.com/security/livepatch
 
+realtime-kernel:
+  help: |
+    Ubuntu's Realtime Kernel is useful for ... TODO
```

Comment:
@nathan-j-hart @EdoardoBarbieriCanonical What should `ua help realtime-kernel` display? See the other service descriptions in this file for examples. It is usually a couple sentences long and often contains a link to more information.

Reply:
Ok we would like it to say this 
>The real-time kernel is a beta version of the 22.04 Ubuntu kernel with the PREEMPT_RT patchset integrated for x86_64 and ARM64. It services extreme latency-dependent use cases and provides deterministic response times to service events. By meeting stringent preemption specifications, the real-time kernel is suitable for telco applications and embedded devices in industrial automation and robotics. To enroll in the beta program, visit https://ubuntu.com/realtime-kernel